app/views/roles/_form.html.erb

```diff
@@ -12,7 +12,25 @@
     <div class="tab-pane active" id="primary">
       <%= text_f f, :name, :class => @role.builtin? ? "disabled" : ""  %>
       <%= hidden_field_tag :original_role_id, @original_role_id if @cloned_role %>
+
+      <% if show_location_tab? || show_organization_tab? %>
+        <h6><%= popover("", _("Whenever organizations or locations role association is changed, the change will propagate to all inheriting filters.
```

Comment:
Text is taking into account the option where both taxonomies are enabled, not one or the other, could you fix that?
The popover looks a bit weird in this case, it's a similar case to the info we show in Provisioning Templates, can you make it like that (an alert-info, or warning if you wish)

![screenshot from 2016-08-25 11-25-46](https://cloud.githubusercontent.com/assets/598891/17963624/ba55cbca-6ab6-11e6-9667-c18e6683bdcd.png)

@domcleal or @GregSutcliffe if you can revise this one too I'd appreciate it